cmd/qumomf/main.go

```diff
@@ -59,7 +62,12 @@ func main() {
 		logger.Warn().Msg("No clusters are found in the configuration")
 	}
 
-	qCoordinator := coordinator.New(logger)
+	db, err := newStorage(cfg)
+	if err != nil {
+		logger.Fatal().Err(err).Msg("failed to init sqlite storage")
```

Comment:
Do not clarify the concrete storage, just 'failed to init persistent storage'. Any sqlite mentions should be in the error message